ckanext/webpageview/tests/test_view.py

```diff
@@ -9,18 +9,19 @@
 
 
 class TestWebPageView(helpers.FunctionalTestBase):
+    @classmethod
+    def _apply_config_changes(cls, cfg):
```

Comment:
Has the logic for loading plugins changed? I can see how `example_idatasetform` would be affected by my first comment in the PR but what about this and other view plugins?